portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,77 @@
     <script src="script.js"></script>
   </head>
   <body>
+
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+      <div>
+        <h1>Sai Sandeep</h1>
+        <h2>Press any key to continue</h2>
+      </div>
+
+      <div>
+        <h2>Hey, I'm Sandeep!</h2>
+        <p>
+          I work with a lot of things, like web, app, and some game development. I also do a bit of data science and
+          machine/deep learning, and participate in competitive programming once in a while. 
+        </p>  
+      </div>
+
+      <div>
+        <h2>
+          Lately, I've been also been trying to learn how to play the guitar.
+        </h2>  
+      </div>
+
+      <div>
+        <h2>
+          ...and learn some Mandarin.
+        </h2>  
+      </div>
+
+      <div>
+        <h2>
+          I am a weekend gamer, mostly into 2D platformers and metroidvanias.
+        </h2>  
+        <p>
+          Some recent games:
+        </p>
+        <ul>
+          <li>Katana ZERO</li>
+          <li>Celeste</li>
+          <li>Sayonara Wild Hearts</li>
+          <li>Hollow Knight</li>
+          <li>Ori and the Blind Forest</li>
+          <li>Gris</li>
+          <li>Enter the Gungeon</li>
+          <li>Dead Cells</li>
+        </ul>
+      </div>
+
+      <div>
+        <h2>
+          Some of my doodles.
+        </h2>  
+      </div>
+
+      <div>
+        <h2>
+          Thank you for playing :)
+        </h2>
+        <ul>
+          <li>
+            <a href="https://www.github.com/heliostrike/" target="_blank">Github</a>
```

Comment:
https://www.w3schools.com/howto/howto_css_social_media_buttons.asp

You can use something like this to have your profiles as buttons.

Reply:
On it. :)